blog/2019-07-12-getting-back-my-domain.md

```diff
@@ -0,0 +1,139 @@
+message proxies. This means that there was no way that I could access the MFA number, and was likely permanently 
+locked out of my Namescheap account. 
+
+This sucked a lot. Not only was I unable to maintain, upgrade, or configure my domain, I risked losing it after my 
+credit card expired and ceased payments. 
+
+The next lever I had left to pull was contacting Namecheap support. In my first conversation with their live help chat 
+service, I saw a glimmer of hope: Their customer service members could override MFA temporarily if I could prove 
+ownership of the account. This glimmer was extinguished soon, however, when I learned that to identify me, they 
+needed the phone number on the account.
+
+With nothing left to do, I filed a ticket at their risk management team, hoping that they could help me. If that failed,
+ my hail mary option was to modify the content of my website, proving that it was really me after all. 
+ 
+A day later, they got back to me. Without the phone number, they could do nothing for me.
+
+* * * 
+
+I have not missed the great, dual irony of the situation: 
+
+- I used a series of indirections to remain anonymous... to publicize myself on the internet. 
+- The security measures I took... made me more vulnerable and less in control.
+
+"Congratulations," [my partner](http://camillemerz.com) told me. "You played yourself."
+
+Thanks, hun.
+
+
+Over the next few days, I searched my email and other records for traces of that number with no success. I knew that 
+this was a futile effort, but I did it anyway out of desperation. 
+
+Then, I had an idea. When I originally created my account, I paid for extra services to make sure that any identifying 
+data about the domain wouldn't be publicly listed. I decided to check that anyway, just in case. 
+
+I ran a query via Whois.org --
+
+"There it is!" I discovered.
+
+Apparently, I had only paid for a year's worth of this service. It had expired long ago. But there it was, a phone 
+number ending in `057`.
+
+Soon after, I had another live chat with Namecheap customer support. Before I knew it, I bypassed the MFA.
+
+* * * 
+
+When I was younger, I had an unbalanced sense of trust. I had strong feelings towards protecting myself online, but no 
+context to correctly model risk. 
+
+It's easy to ask what I could possibly have been thinking back then. I assure you, I have many times during this whole 
+ordeal. 
+
+I had a misplaced estimation of the orders of magnitude of the tactics I exercised. At the time, I didn't know of the 
+importance of certain moves -- like how [important HTTPS is for web browsing](https://doesmysiteneedhttps.com/). I also 
+neglected to consider the longevity of certain moves. I needed to ask myself, "how dependable are the tools I choose to 
+rely on?"
+
+I take away from this experience a lesson in skepticism. Before, I thought I needed to be robust against human 
```

Comment:
What other potential other areas could you improve upon or is it not possible to see yet?